### PR TITLE
MOIS39-ID006 Invite Users to a Household

### PR DESCRIPTION
**Pull request by zlatax**
Completed invite feature. The following are the detailed functionality additions/changes

**List of invites on the homepage**
- Can click the house name to popup more detail on invite
- Can either Accept or Delete Invite
- Shows the number of invites next to Invite Header

**List of Users**
- List of Members of a household are shown on right column of household page
- Displays email if displayName is undefined

**Inviting**
- Users can invite other users to the household by entering their email address and clicking 'ADD'
- It will fail if the user tries to invite themselves, or tries to send an invite to an email address that has already been invited.
- A custom email will be sent to the email address, with a link to the login page of the app. (Invite will automatically appear on homepage after signup)

2 tests are also written
- 01_login: Tests signup, login features
- 02_household: Creating households, inviting other users through email

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -56,16 +66,16 @@ jobs:
         with:
           browser: chrome
           build: npm run build:emulator
-          start: npm run start:emulator
-          wait-on: 'http://localhost:3021'
+          start: npm run dev:emulator
```
Comment (IkePureza) [inline .github/workflows/main.yml:69]: Why are we running cypress with dev instead of build->start?

**Review by IkePureza — APPROVED** [review]
Nice work man! We can close this PR well right #9

File: .github/workflows/main.yml (hunk 1 above)
Comment (zlatax) [reply .github/workflows/main.yml:69]: Cause the instance run in production mode, it wont run tests on the emulators. (Have already tried to change NODE_ENV but couldnt really find a way around it)

**Comment by zlatax** [comment]
Yeah, that was more of a PR for client presentation to create a temporary deployment link, but I'll make the changes you commented on this one and merge this PR. I'll probs close the other one

File: .github/workflows/main.yml (hunk 1 above)
Comment (IkePureza) [reply .github/workflows/main.yml:69]: Ahh I see, yeah all g

**Pull request merged**